android_jni/README.md

```diff
@@ -31,25 +31,25 @@ $ export ANDROID_SDK_ROOT="/path/to/android/sdk"
 $ export ANDROID_NDK_HOME="/path/to/android/ndk"
 ```
 
-Step 3 - Checkout and build libgav1 or dav1d
+Step 3 - Checkout and build dav1d (or libgav1)
 
 ```
 $ cd ext
-$ ./libgav1_android.sh "${ANDROID_NDK_HOME}"
+$ ./dav1d_android.sh "${ANDROID_NDK_HOME}"
 $ cd ..
 ```
 
-If you want to use dav1d instead:
+If you want to use libgav1 instead:
 
 ```
 $ cd ext
-$ ./dav1d_android.sh "${ANDROID_NDK_HOME}"
+$ ./libgav1_android.sh "${ANDROID_NDK_HOME}"
 $ cd ..
 ```
 
 Update [CMakeLists.txt](avifandroidjni/src/main/jni/CMakeLists.txt) as follows:
- * Set `AVIF_LOCAL_LIBGAV1` and `AVIF_CODEC_LIBGAV1` to `OFF`.
- * Set `AVIF_LOCAL_DAV1D` and `AVIF_CODEC_DAV1D` to `ON`.
+ * Set `AVIF_LOCAL_LIBGAV1` and `AVIF_CODEC_LIBGAV1` to `ON`.
+ * Set `AVIF_LOCAL_DAV1D` and `AVIF_CODEC_DAV1D` to `OFF`.
```

Comment:
Nit/optional: Should we reverse these two lines?

Reply:
Done.